
syntax = "proto3";

option cc_generic_services = true;

message AddReq
{
    optional int32 a=1;
    optional int32 b=2;
}

message AddRsp
{
    optional int32 result=1;
}

message EchoReq
{
    optional string str=1;
}

message EchoRsp
{
    optional string str=1;
}



// service Echo
// {
//     rpc EchoService(EchoRsp) returns(EchoRsp);
// }
